code/modules/organs/prosthetics/prosthetics_manufacturer_models.dm

```diff
@@ -10,7 +10,7 @@
 	icon_base = 'icons/mob/human_races/cyberlimbs/morgan/morgan_main.dmi'
 	modifier_string = "wooden"
 	hardiness = 0.75
-	manual_dexterity = DEXTERITY_SIMPLE_MACHINES
+	manual_dexterity = DEXTERITY_SIMPLE_MACHINES | DEXTERITY_HOLD_ITEM | DEXTERITY_EQUIP_ITEM | DEXTERITY_KEYBOARDS | DEXTERITY_GRAPPLE
```

Comment:
i don't know how to feel about that. what does this even prevent you from doing, now? it kind of feels like we're making wooden limbs not actually much worse than modern prostheses, especially given that DEXTERITY_COMPLEX_TOOLS or whatever isn't really used much on shaded hills. maybe we should just make bumping airlocks not check dexterity if it's worn and not held? and to open structure doors, we could make it so that it just needs DEXTERITY_HOLD_ITEM.

alternatively i'm open to hearing why wooden prostheses should have all of these? but I kind of liked the niche they had before.

Reply:
sure, I'll take that. the `name == "Space"` thing should still go though, otherwise that's the only change requested